### PR TITLE
Run iOS functional tests on azure pipelines

### PR DESCRIPTION
**Pull request by ki4070ma**
For https://github.com/appium/python-client/issues/377

### CODE REVIEW

**Comment by ki4070ma** [comment]
### TODO in the next PR or later
* Publish test report
* Run Android tests

@KazuCocoa @mykola-mokhnach 
Can I merge this PR and update along to review comments in the next PR?
Because I can't setup azure pipeline before `azure-pipelines.yml` exists in repository.
Or I'll check the behavior on my forked repository with changes along to review comments.

I've checked this PR behavior on forked repository.
https://dev.azure.com/ki4070ma/ki4070ma.python-client/_build/results?buildId=49

File: ci-jobs/functional/run_appium.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- script: npm install -g appium
+  displayName: Install appium
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.x'
+- script: appium --relaxed-security &
+  displayName: Run Appium in background
+- script: brew install ffmpeg
+  displayName: Resolve dependencies (Appium server)
+- script: python setup.py install
```
Comment (KazuCocoa) [inline ci-jobs/functional/run_appium.yml]: What about calling only `pipenv` like install CI related dependencies by `ci-requirements.txt`?

File: ci-jobs/functional/run_appium.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: Install Node 10.x
+- script: npm install -g appium
+  displayName: Install appium
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.x'
+- script: appium --relaxed-security &
```
Comment (KazuCocoa) [inline ci-jobs/functional/run_appium.yml]: Could you move running appium after all check?

Comment (KazuCocoa) [reply ci-jobs/functional/run_appium.yml]: You can also run appium server via Python script

Comment (ki4070ma) [reply ci-jobs/functional/run_appium.yml]: > You can also run appium server via Python script

Let me have another PR for this. I will think about how to use it and it(`Appium_service`) doesn't work well when I tried on my local..

**Comment by ki4070ma** [comment]
The CI result >  https://dev.azure.com/ki4070ma/ki4070ma.python-client/_build/results?buildId=54

File: ci-jobs/functional/run_appium.yml (hunk 1 above)
Comment (ki4070ma) [reply ci-jobs/functional/run_appium.yml]: Used existing `Pipfile`, it's too enough for ci, though.
`Pipfile` seems impossible to have another section for CI like package/dev-package.

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Xcode
+# Build, test, and archive an Xcode workspace on macOS.
+# Add steps that install certificates, test, sign, and distribute an app, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/xcode
+jobs:
+  - job: func_test_ios1
+    pool:
+      vmImage: 'macOS-10.14'
```
Comment (ki4070ma) [inline azure-pipelines.yml]: (notice) I'd like to have `parameters` section, but can't have it in `azure-pipelines.yml`.

[ref]
https://github.com/appium/ruby_lib_core/blob/master/ci-jobs/functional_test.yml

**Comment by ki4070ma** [comment]
I've set up azure pipelines for https://github.com/appium/python-client .

https://dev.azure.com/ki4070ma/python-client/_build

**Pull request merged**